packages/internals/src/utils/trimBlocksFromSchema.ts

```diff
@@ -18,7 +18,8 @@ export function trimNewLine(str: string): string {
 }
 
 export function trimBlocksFromSchema(str: string, blocks = ['model', 'enum', 'datasource', 'generator']): string {
```

Comment:
What is this used for?

Reply:
Only for db pull to remove the `datasource` block when using --url (if there is a schema locally)

First time I see this!

Reply:
Ugly ugly.